modules_basic/timestamp.js

```diff
@@ -1,28 +1,47 @@
-var h = require('hyperscript')
+var fs = require('fs')
+var Path = require('path')
+var h = require('../h')
 var human = require('human-time')
 
 exports.needs = {}
 
-exports.gives = 'message_meta'
+exports.gives = {
+  timestamp: true,
+  mcss: true
+}
 
 exports.create = function () {
-
-  function updateTimestampEl(el) {
-    el.firstChild.nodeValue = human(new Date(el.timestamp))
-    return el
-  }
-
   setInterval(function () {
```

Comment:
right, this is probably where mutation is being introduced - and needs setImmediate